pkg/backend/display/progress.go

```diff
@@ -165,6 +165,10 @@ func camelCase(s string) string {
 	return string(runes)
 }
 
+func SimplifyTypeName(typ tokens.Type) string {
+	return simplifyTypeName(typ)
```

Comment:
If we're exporting a new API here (even within pkg), consider adding a doc comment specifying what this does and maybe even a unit test.
It's not otherwise obvious what this does without walking through the implementation line by line.